docs/02-membership.md

```diff
@@ -95,7 +95,7 @@ The membership is a set of people working within the eligible projects who have
 | [Michelangelo Riccobene](https://github.com/mriccobene/) | 1 | Erigon | |
 | [Somnath Banerjee](https://github.com/somnathb1/) | 1 | Erigon | |
 | [Tullio Canepa](https://github.com/canepat/) | 1 | | [erigontech/silkworm](https://github.com/erigontech/silkworm/pulls?q=author%3Acanepat) [ledgerwatch/erigon](https://github.com/ledgerwatch/erigon/pulls?q=author%3Acanepat) |
-| [Pooja Ranjan](https://github.com/poojaranjan/) | 1 | Ethereum Cat Herders | |
+| [Pooja Ranjan](https://github.com/poojaranjan/) | 1 | | [ethereum/pm](https://github.com/ethereum/pm/pulls?q=is%3Apr+is%3Aclosed+poojaranjan), [ethereum/eips](https://github.com/ethereum/EIPs/pulls?q=is%3Apr+is%3Aclosed+poojaranjan), [Ethereum Protocol](https://www.youtube.com/@EthereumProtocol)|
```

Comment:
can you modify [Ethereum Protocol] to [Protocol Call Streaming] or [Protocol Call Logistics]?

Reply:
The channel's name is `@EthereumProtocol` with URL as `https://www.youtube.com/@EthereumProtocol`. If it has to be renamed for clarity, maybe `EthereumProtocol Videos`?

Reply:
sounds good!